### ISSUE TITLE
Problem: Face image generation with StyleGAN generates darker and darker output.

### ISSUE THREAD

**Issue by notnot** [keras-team/keras-io#830]
Problem: Face image generation with StyleGAN generates darker and darker output.
I am running the stylegan.py example, doing a training, and the more steps_per_epochs i use, the darker the output becomes, For instance, with steps_per_epoch = 2048, all the output is black. Something is wrong. I did not change anything in the example code except increasing steps_per_epoch in the given train() function. 

The output that is generated by loading the pre-trained model is ok, as shown on https://keras.io/examples/generative/stylegan/

**Comment by notnot** [comment]
I found the bug in stylegan.py:
The input images are 'normalized' twice, so that they become very dark. 

The first normalization is done here:

ds_train = ds_train.map(lambda x: x / 255.0)

the second (in resize_image) by this:

image = tf.cast(image, tf.float32)/127.5 - 1.

The latter one is correct, and should be the only one. 
Delete the ' ds_train = ds_train.map(lambda x: x / 255.0)'  line to fix the bug.
Interestingly, the GAN was working, but unexpectedly learning to output very very dark images ;)

**Comment by BernardoOlisan** [comment]
they never answer in here bro
:/

**Comment by notnot** [comment]
@LukeWood @fchollet @yarri-oss 
Please integrate the StyleGAN bug fix mentioned above.

**Comment by yarri-oss** [comment]
LGTM but would ask @soon-yau to confirm as well because I'm not sure I tested agains the correct public version of Celeb A

**Comment by soon-yau** [comment]
@yarri-oss the "ds_train = ds_train.map(lambda x: x / 255.0)"  was wrongly inserted in your last commit. You can create a PR to remove it.

**Comment by sonali-kumari1** [comment]
Hi @notnot - 
This issue was addressed in PR [#1358](https://github.com/keras-team/keras-io/pull/1358). Please feel free to close this issue if there are no further concerns. Thanks!
